tests/paysafe/DirectDebitServiceTest.php

```diff
@@ -0,0 +1,505 @@
+<?php
+/**
+ * TODO appropriate copyright notice
+ */
+
+namespace Paysafe;
+
+/*
+ * This class provides incomplete coverage of DirectDebitService. I'm only writing today to test splitypay for ACH/EFT
+ * TODO complete coverage
+ */
+
+use function json_decode;
+use function json_encode;
+use Paysafe\DirectDebit\Purchase;
+use PHPUnit_Framework_Error;
+
+class DirectDebitServiceTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var \PHPUnit_Framework_MockObject_MockObject $mock_api_client */
+    private $mock_api_client;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->mock_api_client = $this->createMock(PaysafeApiClient::class);
+        $this->mock_api_client->method('getAccount')->willReturn('bogus_account_num');
+    }
+
+    /**
+     * This is a bad test as it simply confirms current undesirable behavior. If no type is specified in the Purchase
+     * object (ach, eft, etc), then a PHP Error is generated. Ideally, the code would gracefully handle this situation.
+     * See: https://github.com/paysafegroup/paysafe_sdk_php/issues/13
+     */
+    public function testSubmitNoTypeSpecified()
+    {
+        $this->expectException(PHPUnit_Framework_Error::class);
+        $this->expectExceptionMessage('Undefined variable: return');
+        /*
```

Comment:
Note, this and a few other tests make reference to https://github.com/paysafegroup/paysafe_sdk_php/issues/13